crowdgezwitscher/events/views.py

```diff
@@ -1,14 +1,44 @@
 from rest_framework import generics
 
 from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.forms import ModelForm, ModelMultipleChoiceField
 from django.views.generic.list import ListView
 from django.views.generic.detail import DetailView
 from django.views.generic.edit import CreateView, UpdateView, DeleteView
 from django.urls import reverse_lazy
 
+from crowdgezwitscher.models import MapObjectFilter
+from crowdgezwitscher.widgets import SelectizeSelectMultiple
 from events.models import Event
 from events.serializers import EventSerializer, EventSerializerShortened
-from crowdgezwitscher.models import MapObjectFilter
+from facebook.models import FacebookPage
+
+
+class EventForm(ModelForm):
```

Comment:
Nur zum Verständnis: Seh ich das richtig, dass der ganze Aufwand in `EventForm` nur so groß ist, weil die M2M-Beziehung als Attribut `events` an einer `FacebookPage` hängt statt an `Event` selbst? Das macht `FacebookPageForm` so viel einfacher?
Das wäre ja kaum zu glauben, dass so ein Aufwand nötig ist, wenn man die M2M-Beziehung (bei der die Richtung zwischen den Models ja eigentlich egal ist) nur in einer Richtung leicht nutzen kann und die andere Richtung solch einen Krampf erfordert.